index.md

```diff
@@ -37,3 +37,50 @@ In this article, the authors explain how they use multi-agent reinforcement lear
+
+#### Data Details:
+
+Since this project leverages the use of a genetic neural network (GNN), training is initialized with a random set of parameters that are fine-tuned over the specified training course. 
+
+Thus, we will not require an initial dataset for this project. Rather, we aim to explore how different networks perform when training is initialized with a random parameter set.
+
+#### Project Details:
+
+As described in the Software Details section, the *AI for Snake Game* project we are building on includes a fully-funtional GNN capable of training an agent to play the snake game. Our details below reflect the implementation of this network, but we fully intend to implement and compare multiple networks' performance over the course of this project.
+
+- Type of NN(s) in Use: 
+    
+    This project leverages the use of a GNN comprised of a separate genetic algorithm (GA) and NN. This functions by training snake agents via a 4 layer (1 input, 2 hidden, 1 output) NN. 
+    
+    At the end of a training epoch, the set of *n* agent results is fed to a GA that improves agent performance through the following four methods (note that Fitness Calculations occur once per epoch, and Selection, Crossover, and Mutation are performed sequentially *n*/2 times):
+
+    - Fitness Calculations:  Calculations that assign scores to each agent based on its performance during the training epoch
+    - Selection: The algorithm selects 2 high-performing agents to be used for Crossover
+    - Crossover: The algorithm mixes characteristics of each relatively successful parent agent to create a new child agent
+    - Mutation: The algorithm randomly mutates aspects of the child agent's characteristics in order to prevent training stagnation
+
+    After the above steps are performed, the GA creates a set consisting of the top 50% of agents from the initial population and the newly generated child population and propogates them to the following training epoch. This process repeats for a specified number of epochs that each contain gameplay for a specified number of unique agents.
+
+- NN Input Details:
+    - The [Manhattan distance](https://xlinux.nist.gov/dads/HTML/manhattanDistance.html) between the snake agent's head and the game board fruit
+    - Available space on the grid in each of the four available directions (left, right, up, down) from the snake agent's head
+    - The current length of the snake agent
+
+- NN Ouput Details: 
+    - This network has four outputs (representing the four available movement directions) ranging in value from 0 to 1 where
+    - The snake agent will move in the direction of the output with the highest associated value
+
```

Comment:
Is any of this writing copied directly from the original repo? If so, just make sure to add direct quotes. Regardless, it might be nice to add a little blurb at the end of this section that a lot of the information is coming from his website.

Reply:
None is directly copied, but good idea on adding a direct reference to our boi Craig